docs/concepts/MSBuild-props-and-targets.md

```diff
@@ -0,0 +1,74 @@
+---
+title: MSBuild props and targets in a package
+description: Describes MSBuild props and targets in NuGet packages
+author: nkolev92
+ms.author: nikolev
+ms.date: 07/13/2022
+ms.topic: conceptual
+---
+
+# MSBuild props and targets in a package
+
+In additional to the more traditional assemblies, NuGet packages may sometimes add custom build targets or properties to projects that consume that package.

```

Comment:
I suggest using .props and .targets instead of props and targets, respectively, since this is how it appears to be in MSBuild docs: https://docs.microsoft.com/visualstudio/msbuild/customize-your-build?view=vs-2022#choose-between-adding-properties-to-a-props-or-targets-file

Example below.

```suggestion
In additional to the more traditional assemblies, NuGet packages can add custom MSBuild `.targets` files or `.props` files to projects consuming the package.
```

Reply:
Fixed. all through the doc.